docs-markdown/src/controllers/media-controller.ts

```diff
@@ -261,15 +261,15 @@ export function Insert(isArt: any) {
         // Determine if there is selected text.  If selected text, no action.
         if (isArt && selectedText === "") {
             vscode.window.showInputBox({
-                placeHolder: "Add alt text (up to 70 characters)",
+                placeHolder: "Add alt text (up to 250 characters)",
             }).then((val) => {
                 if (!val) {
                     getFilesShowQuickPick(isArt, "");
                     vscode.window.showInformationMessage("No alt entered or selected.  File name will be used.");
                 } else if (val.length < 70) {
                     getFilesShowQuickPick(isArt, val);
                 } else if (val.length > 70) {
```

Comment:
@bharney0 , looks good.  I think we might just need to bump the length evaluations too and everything will be good to go.

Reply:
@jamarw good call. Thanks for catching that. The code has been updated to us 250 instead of 70.